x/pricefeed/simulation/operations.go

```diff
@@ -198,3 +129,93 @@ func getExpiryTime(ctx sdk.Context) (t time.Time) {
 	// need to use the blocktime from the context as the context generates random start time when running simulations
 	return ctx.BlockTime().Add(time.Second * 1000000)
 }
+
+// PriceGenerator allows deterministic price generation in simulations
+type PriceGenerator struct {
```

Comment:
I'd move this struct and its functions to a new file `x/pricefeed/simulation/types.go` to keep logic clean from type definitions 👍

Reply:
👍  to moving the type out

Reply:
Great idea 💯